### ISSUE TITLE
`statement_position` rule customization

### ISSUE THREAD

**Issue by andreiardelean-kwiff** [realm/SwiftLint#3660]
I would like `statement_position` to accept: 

```swift
if condition { print("\(value1)") }
else { print("\(value2)") }
```
```swift
if condition { 
    print("\(value1)") 
} else { 
    print("\(value2)") 
}
```
but don't accept :
```swift
if condition { 
    print("\(value1)") 
} 
else { 
    print("\(value2)") 
}
```

Basically, allow `\n else` only if the `{` and `}` are on the same line with `else`.

**Comment by lhunath** [comment]
I came here to ask for the exact opposite; ie. a configuration for `statement_position` to ensure `else`, `catch` are styled as:

```
if condition { 
    print("\(value1)") 
} 
else { 
    print("\(value2)") 
}
```

since they are keywords introducing a new block, it's more symmetric to have them formatted as equivalent. In this context, the following would also be permissible:

```
if condition { print("\(value1)") }
else { print("\(value2)") }
```

Essentially the rule would be, every block is introduced by a keyword, and the keyword should be the first thing on the line, not obscured behind something else.

**Comment by thisIsTheFoxe** [comment]
@lhunath I'm pretty sure your request is the same as #1181 and the configuration for that i.e. `uncuddled_else` exists already.
